geode-core/src/main/java/org/apache/geode/distributed/AbstractLauncher.java

```diff
@@ -157,7 +162,7 @@ protected static Properties loadGemFireProperties(final URL url) {
     if (url != null) {
       try {
         properties.load(new FileReader(new File(url.toURI())));
-      } catch (Exception e) {
+      } catch (Exception ignored) {
```

Comment:
Are we really ignoring the exception if we're doing something in response to it?
  Ditto lines 272, 438.